### DISCUSSION TITLE
24.1.0

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1673] [General]
## Releases Notes for 24.1.0
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.1.0/draw.io-24.1.0-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.1.0/draw.io-24.1.0-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v24.1.0/draw.io-universal-24.1.0.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v24.1.0/drawio-amd64-24.1.0.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v24.1.0/drawio-x86_64-24.1.0.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v24.1.0/drawio-x86_64-24.1.0.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- #1504 
- Updates to [draw.io core 24.1.0](https://github.com/jgraph/drawio/blob/v24.1.0/ChangeLog). All changes from 24.0.5 and 24.1.0 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v24.1.0'>24.1.0</a>.</em>

**Comment by josephowr** [comment]
I was experiencing problems when deleting the current version.

**Comment by jabicego** [comment]
Excellent software, congrats!
